internal/completions/client/awsbedrock/bedrock.go

```
@@ -282,6 +271,28 @@ func (c *awsBedrockAnthropicCompletionStreamClient) makeRequest(ctx context.Cont
 	return resp, nil
 }
 
+// Builds a bedrock api URL from the configured endpoint url.
+// If the endpoint isn't valid, falls back to the default endpoint for the specified fallbackRegion
+func buildApiUrl(endpoint string, model string, stream bool, fallbackRegion string) *url.URL {
+	apiURL, err := url.Parse(endpoint)
+	if err != nil || apiURL.Scheme == "" {
```

Do we want to just swallow the error here? Could be hard to debug why requests are being made to the wrong region 🤔

---

This is how it worked before, I've not changed any functionality

---

I think the idea is that it should handle blank config settings etc as well